setup.py

```diff
@@ -24,7 +24,7 @@
     "pyasn1-modules>=0.2.1",
     # rsa==4.5 is the last version to support 2.7
     # https://github.com/sybrenstuvel/python-rsa/issues/152#issuecomment-643470233
-    'rsa<4.6; python_version < "3.6"',
+    'rsa<4.9; python_version < "3.6"',
```

Comment:
Nope.  4.6 was the last Python2-compatible release.  The 2.7 tests are passing because `testing/constraints-2.7.txt` pins the correct version.